docs/developer-guide/api.rst

```diff
@@ -227,18 +227,84 @@ The KSQL resource runs a sequence of KSQL statements. All statements, except tho
           "commandStatus": {
             "status":"SUCCESS",
             "message":"Stream created and running"
-          }
+          },
+          "commandSequenceNumber":10
         },
         {
           "statementText":"CREATE STREAM pageviews_alice AS SELECT * FROM pageviews_original WHERE userid='alice';",
           "commandId":"stream/PAGEVIEWS_ALICE/create",
           "commandStatus": {
             "status":"SUCCESS",
             "message":"Stream created and running"
-          }
+          },
+          "commandSequenceNumber":11
         }
       ]
 
+.. _coordinate_multiple_requests
+
+Coordinate Multiple Requests
+----------------------------
+
+To submit multiple, interdependent requests, there are two options. The first is to submit them as a single request,
+similar to the example request above:
```

Comment:
```suggestion
similar to the example request below:
```

Reply:
Oh I actually meant above (see lines 201-214). Normally I'd throw in a link but it feels a little overkill since it really is the content directly above this sentence. If a link is preferred I can add it though.

Reply:
oh... if it looks good compiled then you can ignore this